3회차/신윤식/4_신용카드만들기1.py

```diff
@@ -1,3 +1,18 @@
 import sys
 
 sys.stdin = open("_신용카드만들기1.txt")
+
+T = int(input())
+
+for i in range(1, T+1):
+    card_num15 = list(map(int,input().split()))
+    odd_number = card_num15[::2]
+    even_number = card_num15[1::2]
+    for j in range(len(odd_number)):
+        odd_number[j] *= 2
+    sum_oe = sum(odd_number) + sum(even_number)
```

Comment:
`odd_number`의 원소들을 2배 씩 해주지 않고, 마지막에 전체 합만 2배 해주는 방법도 있습니다! 
```suggestion
    sum_oe = sum(odd_number) * 2 + sum(even_number)
```